src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -16,7 +16,12 @@ def get_link(config, filepath, default, version=None):
 
     formatted_path = filepath.replace(config.stack_root, config.source_root, 1)
 
-    return install.get_stacktrace_link(config.repository, formatted_path, default, version)
+    attempted_url = None
+    link = install.get_stacktrace_link(config.repository, formatted_path, default, version)
+
+    if not link:
+        attempted_url = install.format_source_url(config.repository, formatted_path, default)
```

Comment:
@MeredithAnya why do we use `default` here instead of the version or default?

Reply:
> We don't need to use the `version` (aka the commit) because when we actually call the API to check the file, if we fail with the commit, we fall back to the default already.